packages/react-sdk/src/locales/de/neoboard.json

```diff
@@ -140,7 +140,7 @@
       "content": "<p>Du kannst das NeoBoard und damit die geleistete Arbeit exportieren und (bspw. in einem anderen Raum) wieder importieren oder als Sicherungskopie ablegen. Außerdem kannst du das Raster (de-)aktivieren.</p>",
       "title": "Einstellungen"
     },
-    "slideCount": "{{index}} von {{total}}",
+    "slideCount": "",
```

Comment:
might cause an error if we remove its use (see above)

Reply:
won't fix (see above)